src/Core/Commander/Providers/TileProvider.js

```diff
@@ -43,28 +37,26 @@ TileProvider.prototype.executeCommand = function executeCommand(command) {
         bbox,
         level: (command.level === undefined) ? (parent.level + 1) : command.level,
         segment: 16,
-        parentMaterial: parent.material,
-        parentWmtsCoords: parent.wmtsCoords,
+        parentMaterial: parent ? parent.material : null,
+        parentWmtsCoords: parent ? parent.wmtsCoords : null,
     };
 
-    const geometry = new TileGeometry(params, this.builder);
+    const geometry = new TileGeometry(params, command.layer.builder);
 
-    var tile = new command.type(geometry, params);
+    var tile = new TileMesh(geometry, params);
 
-    tile.setUuid(this.nNode++);
-    tile.link = parent.link;
+    tile.layer = command.layer.id;
+    tile.layers.set(command.threejsLayer);
+    tile.setUuid();
     tile.geometricError = Math.pow(2, (18 - params.level));
 
-    parent.worldToLocal(params.center);
+    if (parent) {
+        parent.worldToLocal(params.center);
+    }
 
     tile.position.copy(params.center);
     tile.setVisibility(false);
-
-    parent.add(tile);
     tile.updateMatrix();
-    tile.updateMatrixWorld();
-    tile.OBB().parent = tile;   // TODO: we should use tile.add(tile.OBB())
-    tile.OBB().update();
 
     // update bbox if node herits texture elevation from parent
     if (tile.material.getElevationLayerLevel() > EMPTY_TEXTURE_ZOOM) {
```

Comment:
to patch this :
https://github.com/iTowns/itowns2/pull/239/files#diff-e729b37b590009b20e3b637dcc65321fR68

add this 

```js
if (tile.material.getElevationLayerLevel() > EMPTY_TEXTURE_ZOOM) {
    const textureElevation = tile.material.getLayerTextures(l_ELEVATION)[0];
    // need to maximize elevation when delta zoom is important
    const isBigDelta = tile.level - tile.material.getElevationLayerLevel() > 6;

    const { min, max } = !isBigDelta ? OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
            textureElevation.image.data,
            SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
            tile.material.offsetScale[0][0]) : {
                min: parent.bbox.minCoordinate.altitude(),
                max: parent.bbox.maxCoordinate.altitude(),
            };

    if (min && max) {
        tile.setBBoxZ(min, max);
    }
}
```